src/script/page/MainContent/panels/preferences/AccountPreferences.tsx

```diff
@@ -165,7 +165,7 @@ const AccountPreferences: React.FC<AccountPreferencesProps> = ({
           >
             <NameInput {...{canEditProfile, name, userRepository}} />
             <UsernameInput {...{canEditProfile, userRepository, username}} domain={showDomain ? domain : undefined} />
-            {email && <EmailInput {...{canEditProfile, email, userRepository}} />}
+            {email && !selfUser.isNoPasswordSSO && <EmailInput {...{canEditProfile, email, userRepository}} />}
```

Comment:
do we really want to completely hide the email input? Don't we want to just disable it (but still display it)?